shellwhat/checks.py

```diff
@@ -1,8 +1,19 @@
 import re
-from functools import partial
+from functools import partial, wraps
 from protowhat.checks.check_logic import *
 from protowhat.checks.check_simple import *
 
+ANSI_REGEX = "(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]"
+
+def _strip_ansi(result):
+    return re.sub(ANSI_REGEX, '', result)
+
+def strip_ansi(state):
```

Comment:
You're only creating this for the sake of the test? Maybe just update the test a bit? Just a suggestion, you know better.